lib/Runtime/Library/JavascriptPromise.h

```diff
@@ -470,6 +472,7 @@ namespace Js
     protected:
         Field(PromiseStatus) status;
         Field(Var) result;
+        Field(bool) isHandled;
```

Comment:
nit: move this up above var just to get better packing? instead of 40 bytes it can be in 32 bytes.

Reply:
Fixed